Simplenote/src/main/res/values/dimens.xml

```diff
@@ -44,6 +44,7 @@
     <dimen name="corner_radius">4dp</dimen>
     <dimen name="divider_height">1dp</dimen>
     <dimen name="elevation_bar">8dp</dimen>
+    <dimen name="navigation_elevation">16dp</dimen>
```

Comment:
This value is to reuse the same elevation from the material library: https://cs.android.com/android-studio/platform/tools/adt/idea/+/mirror-goog-studio-master-dev:designer/testData/property/material.xml;l=379?q=design_navigation_elevation&sq=